Core/Inc/cockpit/feedback.h

```diff
@@ -0,0 +1,73 @@
+#ifndef FEEDBACK_H
+#define FEEDBACK_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "eagletrt.h"
+
+/*!
+ * \brief Possible return codes for feedback functions
+ */
+enum FeedbackState {
+    FEEDBACK_STATUS_LOW,  //!< Feedback line is low
+    FEEDBACK_STATUS_HIGH, //!< Feedback line is high
+    FEEDBACK_STATUS_ERROR //!< Error in reading feedback button states
+};
+
+/*!
+ * \brief Enum for identifying feedback lines, the names will be determined later when hardware is finalized
+ */
+enum FeedbackName {
+    FEEDBACK_NAME_1,    //!< Feedback line 1
+    FEEDBACK_NAME_2,    //!< Feedback line 2
+    FEEDBACK_NAME_3,    //!< Feedback line 3
+    FEEDBACK_NAME_COUNT //!< Total number of feedback lines
+};
+
+/*!
+ * \brief Possible return codes for feedback initialization function
+ */
+enum FeedbackReturnCode {
+    FEEDBACK_RC_ERROR, //!< Error during initialization
+    FEEDBACK_RC_OK     //!< Initialization successful
+};
```

Comment:
Usually the value `0` is intended as "_success_" so the `OK` return code is usually put at the top of the enum.
Not really an error, just a common thing between codebases.
```suggestion
/*!
 * \brief Possible return codes for feedback initialization function
 */
enum FeedbackReturnCode {
    FEEDBACK_RC_OK,     //!< Initialization successful
    FEEDBACK_RC_ERROR //!< Error during initialization
};
```